server/server_test.go

```diff
@@ -339,58 +339,7 @@ func TestSendSecretHandler(t *testing.T) {
 		assertStatusCode(t, http.StatusCreated, response.Code)
 	})
 
-	t.Run("request missing content-type header", func(t *testing.T) {
-		req, err := http.NewRequest("POST", "/v1/secrets", nil)
-		assert.ErrorIsNil(t, err)
-
-		response := httptest.NewRecorder()
-		subrouter.ServeHTTP(response, req)
-
-		assertStatusCode(t, http.StatusBadRequest, response.Code)
-		assertHasJSONErrorDetail(t, response.Body,
-			"expecting header Content-Type: application/json")
-	})
-
-	t.Run("request content-type header isn't application/json", func(t *testing.T) {
-		req, err := http.NewRequest("POST", "/v1/secrets", nil)
-		assert.ErrorIsNil(t, err)
-
-		req.Header.Set("Content-Type", "multipart/form-data")
-
-		response := httptest.NewRecorder()
-		subrouter.ServeHTTP(response, req)
-
-		assertStatusCode(t, http.StatusBadRequest, response.Code)
-		assertHasJSONErrorDetail(t, response.Body,
-			"expecting header Content-Type: application/json")
-	})
-
-	t.Run("empty request body", func(t *testing.T) {
-		req, err := http.NewRequest("POST", "/v1/secrets", nil)
-		assert.ErrorIsNil(t, err)
-
-		req.Header.Set("Content-Type", "application/json")
-
-		response := httptest.NewRecorder()
-		subrouter.ServeHTTP(response, req)
-
-		assertStatusCode(t, http.StatusBadRequest, response.Code)
-		assertHasJSONErrorDetail(t, response.Body, "empty request body")
-	})
-
-	t.Run("invalid JSON", func(t *testing.T) {
-		req, err := http.NewRequest("POST", "/v1/secrets", strings.NewReader("invalid json"))
-		assert.ErrorIsNil(t, err)
-
-		req.Header.Set("Content-Type", "application/json")
-
-		response := httptest.NewRecorder()
-		subrouter.ServeHTTP(response, req)
-
-		assertStatusCode(t, http.StatusBadRequest, response.Code)
-		assertHasJSONErrorDetail(t, response.Body,
-			"invalid JSON: invalid character 'i' looking for beginning of value")
-	})
+	testEndpointRejectsBadJSON(t, "POST", "/v1/keys")
```

Comment:
What's the signifiance of having this line appear a second time in this test file?

Reply:
One is under `TestUpsertPublicKeyHandler`, the other is under `TestSendSecretHandler` (too many tests in one file + Github UI fail)